site/data/core-team.yml

```diff
@@ -27,3 +27,6 @@
 
 - name: Shohei Yoshida
   user: ysds
+
+- name: Gaël Poupard
+  user: ffoodd
```

Comment:
Missing EOF line

Reply:
doesn't look like the original had one, unless i'm missing something?

Reply:
added one now though

Reply:
It did have one :)

PS. please jump on Slack

Reply:
yup, seems i missed it somehow, disregard :)